examples/airgap/main.tf

```diff
@@ -0,0 +1,98 @@
+locals {
+  vpc_id         = "vpc-aabbccddeeffgghhiijj"
+  tag            = "tfe"
+  setup_bucket   = "mycompany-tfe-setup"
+  domain         = "mycompany-internal.net"
+  hostname       = "tfe"
+  license_file   = "mylicense.rli"
+  airgap_package = "v201911-1.airgap"
+  region         = "us-west-2"
+}
+
+provider "aws" {
+  region = "${local.region}"
+}
+
+module "external" {
+  source  = "hashicorp/terraform-enterprise/aws/modules/external-services"
+  version = ">= 0.0.5-beta"
+
+  vpc_id     = "${local.vpc_id}"
+  install_id = "${module.terraform-enterprise.install_id}"
+
+  rds_subnet_tags = {
+    "Usage" = "${local.tag}"
+  }
+}
+
+resource "aws_iam_role_policy" "setup-bucket" {
+  role = "${module.terraform-enterprise.iam_role}"
+  name = "${local.setup_bucket}-${module.terraform-enterprise.install_id}"
+
+  policy = <<__policy
```

Comment:
this isn't a blocker, but for ease of not having to write json, what do we think with regards to wiriting this as an `aws_iam_policy_document` data object?

Reply:
Oh yeah, probably should do that.